helm/observability-policies/templates/NOTES.txt

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.enabled }}
+The {{ .Chart.Name }} chart has been deployed.
+
+This chart installs Kyverno policies to prevent the accidental deletion of critical CustomResourceDefinitions (CRDs) related to observability.
+
+Protected CRDs:
+{{- range .Values.resources }}
+- {{ . }}
+{{- end }}
+
+To verify the policies:
```

Comment:
I'd rather have the doc in a public place (like this repo's README) and the install provide a link to the doc.